workdir/app-near/src/context.h

```diff
@@ -7,13 +7,13 @@
 
 // A place to store information about the transaction
 // for displaying to the user when requesting approval
-// 44 for address/id and +1 for \0
+// 64 for address/id and +1 for \0
 typedef struct uiContext_t {
-	char line1[45];
```

Comment:
@volovyk-s do all of these have to be increased? Asking as Ledger has super limited RAM.

Make sure to check stack doesn't overflow:
https://ledger.readthedocs.io/en/latest/additional/security_guidelines.html#stack-overflow

Reply:
Check this out as well on how to check for available stack space:
https://github.com/Zilliqa/ledger-app-zilliqa/blob/8cb6e44c810e704aebefc3b84a3d68ed4a141f93/src/zilliqa.c#L351

Reply:
Yes, we need to increase the size of line2 and line3 only, good point.

Ledger has ~700 bytes of call stack memory and 4Kb of RAM in total. Changes in this PR affect RAM, not the call stack. HAVE_BOLOS_APP_STACK_CANARY flag and the logic that is used by Zilliqa help us to control usage of the call stack only, I have added this code so we can control it in the future:   https://github.com/near/app-near/pull/8

RAM usage is ok.